libraries/ESP8266SSDP/ESP8266SSDP.cpp

```diff
@@ -295,6 +295,22 @@ void SSDPClass::schema(WiFiClient client) {
                );
 }
 
+void SSDPClass::schema(Print &print) const {
```

Comment:
This looks very similar to the WiFiClient overload. Why have both? Can they be unified?

Reply:
Done, only ::schema(Print&) survives.